cafes/problem/dton.hpp

```diff
@@ -177,7 +160,7 @@ struct convergence_context2
             KSP ksp;
             PetscInt kspiter;
             PetscReal kspresnorm;
-            std::size_t scale_ = 4;
+            std::size_t scale_ = 1;
```

Comment:
Could you let 4 ?

Reply:
Well, I do not know. I do not see the point to let 4 if it works with 1. But may be there are some particular cases where we need more points on the surface of the particles. What was the reason to put 4 in the first place ?

Reply:
let 1, then !